test/integration-config-mlapiadapter.json

```diff
@@ -1,7 +1,7 @@
 {
   "PORT": 3000,
   "HOSTNAME": "http://ml-api-adapter",
-  "ENDPOINT_SOURCE_URL": "http://central-ledger-int:3001/participants/{{fsp}}/endpoints",
+  "ENDPOINT_SOURCE_URL": "http://central-ledger-int:3001",
```

Comment:
why was this endpoint changed

Reply:
Per troubleshooting session with Miguel where CircleCI integration tests were breaking this was necessary.